From 651b16e6b0ab5812450b60f953616ed82cc4187b Mon Sep 17 00:00:00 2001
From: Theodore Olsauskas-Warren <sauski@google.com>
Date: Fri, 4 Apr 2025 08:09:47 -0700
Subject: [PATCH] update-rand-core

TODO(crbug.com/408276016): Upstream this version bump into the group
repository.

---
 .../rust/chromium_crates_io/vendor/group-v0_13/Cargo.toml       | 2 +-
 .../rust/chromium_crates_io/vendor/group-v0_13/rust-toolchain   | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/third_party/rust/chromium_crates_io/vendor/group-v0_13/Cargo.toml b/third_party/rust/chromium_crates_io/vendor/group-v0_13/Cargo.toml
index 059f0f76146a3..270336ad54355 100644
--- a/third_party/rust/chromium_crates_io/vendor/group-v0_13/Cargo.toml
+++ b/third_party/rust/chromium_crates_io/vendor/group-v0_13/Cargo.toml
@@ -35,7 +35,7 @@ optional = true
 default-features = false
 
 [dependencies.rand_core]
-version = "0.6"
+version = "0.9"
 default-features = false
 
 [dependencies.rand_xorshift]
diff --git a/third_party/rust/chromium_crates_io/vendor/group-v0_13/rust-toolchain b/third_party/rust/chromium_crates_io/vendor/group-v0_13/rust-toolchain
index 3ebf789f5a8df..af92bdd9f58d1 100644
--- a/third_party/rust/chromium_crates_io/vendor/group-v0_13/rust-toolchain
+++ b/third_party/rust/chromium_crates_io/vendor/group-v0_13/rust-toolchain
@@ -1 +1 @@
-1.56.0
+1.63.0
-- 
2.49.0.504.g3bcea36a83-goog

